portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -20,13 +20,85 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+
+import java.util.*;
+import com.google.gson.Gson;
+
+
+
 /** Servlet that returns some example content. TODO: modify this file to handle comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+//   List<String> commentList = new ArrayList<String>();
+
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    Query query = new Query("Comment");
+
+    DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
+    PreparedQuery results = datastore.prepare(query);
+
+    List<String> commentList = new ArrayList<String>();
+
+    for (Entity entity : results.asIterable()) {
+      String firstName = (String) entity.getProperty("firstName");
+      String lastName = (String) entity.getProperty("lastName");
+      String comment = (String) entity.getProperty("comment");
+      String commentItem = firstName + " " + lastName + " : " + comment;
+      commentList.add(commentItem);
+    }
+
+    String json = convertToJson(commentList);
+    response.setContentType("application/json;");
+    response.getWriter().println(json);
+  }
+
+  private String convertToJson(List<String> list) {
+    Gson gson = new Gson();
+    String json = gson.toJson(list);
+    return json;
+  }
+
+  @Override
+  public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String firstName = getParameter(request, "firstname", "");
+    String lastName = getParameter(request, "lastname", "");
+    String comment = getParameter(request, "comment", "");
+    
+    // String commentItem = getParameter(request, "firstname", "") + " " + getParameter(request, "lastname", "") + " : " + getParameter(request, "comment", "");
```

Comment:
Remove commented lines.

Reply:
left commented lines as vestiges of previous PRs, sorry. fixed